examples/notebooks/mnist.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
yeah it runs correctly now -- its too slow to run on a non-gpu node though.  I can clear the errors